torch/_subclasses/meta_utils.py

```diff
@@ -92,6 +94,20 @@ def assert_eq(a: _T, b: _T) -> None:
     assert a == b, f"{a} != {b}"
 
 
+DISABLE_INFERENCE_MODE = False
```

Comment:
nit: can we make this a real config somewhere, just in case someone actually needs to flip it if we end up breaking inference code in a subtle way.

I guess the annoying part is that we don't have global config for fake tensor... Maybe in dynamo config?

Reply:
Added a config. And the behavior is controlled in `convert_frame.py`